recipes/fast-cdr/all/patches/1.0.21-0001-add-cppstd.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 114c3ba..31b1583 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -82,6 +82,8 @@ if(CMAKE_VERSION VERSION_LESS 3.0)
+ else()
+     cmake_policy(SET CMP0048 NEW)
+     project(fastcdr VERSION "${LIB_VERSION_STR}" LANGUAGES C CXX)
++    set(CMAKE_CXX_STANDARD 11)
++    set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
```suggestion
```

Remove this patch, send a PR to the Upstream if possible.
You can add that configuration directly in the cmake wrapper.

Reply:
Should this not be configured by Conan? if the user sets it to CXX 14 it should still compile?

It more of a 
https://github.com/conan-io/conan-center-index/blob/5788f3dbf49d11b917f38d091016119b49ec3084/recipes/zstr/all/conanfile.py#L22-L23